edge-util/src/Microsoft.Azure.Devices.Edge.Util/CertificateHelper.cs

```diff
@@ -208,6 +209,12 @@ public static async Task<(X509Certificate2 ServerCertificate, IEnumerable<X509Ce
             return ParseCertificateResponse(response);
         }
 
+        public static async Task<IEnumerable<X509Certificate2>> GetTrustBundleFromEdgelet(Uri workloadUri, string workloadApiVersion, string moduleId, string moduleGenerationId)
```

Comment:
I am not sure this is a good idea.. The idea for this Helperfile is to contain only static cert related helpers. It should probably not have any workload api/edgelet related APIs.

Reply:
There is prior precedent in this file to obtain the Edge Hub certificate and key. Perhaps this needs to be refactored as a whole. We can do this in a separate PR.